app/views/decidim/action_delegator/admin/manage_delegations/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_tag(setting_manage_delegations_path, multipart: true, class: "form new_import") do %>
+  <%= hidden_field_tag :setting_id, current_setting.id %>
+  <div class="card">
+    <div class="card-divider">
+      <h2 class="card-title">
+        <%= t(".title") %>
+      </h2>
+    </div>
+  </div>
+  <p><%= t(".upload_instructions") %></p>
+  <p><%= t(".required_fields") %></p>
+  <p><%= t(".title_example") %></p>
+  <pre class="code-block">
+    granter_email,grantee_email
```

Comment:
```suggestion
    from,to
```
I have the feeling that the words `granter` and `grantee` are very confusing. I would like to know your opinion on this. Do you thing that something simpler like `from`, `to` would simplify this?

Reply:
It's ok for me to make that change, it's easier to understand.